src/robot/parsing/lexer/blocklexers.py

```diff
@@ -233,7 +232,7 @@ def handles(self, statement):
 
     def lexer_classes(self):
         return (InlineIfLexer, IfHeaderLexer, ElseIfHeaderLexer, ElseHeaderLexer,
-                ForLexer, TryLexer, EndLexer, ReturnLexer, KeywordCallLexer)
+                ForLexer, TryLexer, EndLexer, ReturnLexer, KeywordCallLexer, ContinueLexer, BreakLexer)
```

Comment:
I'm not absolutely sure about this, but I believe these new lexers should be before `KeywordCallLexer` here and in other `lexer_classes`. Otherwise `BREAK` and `CONTINUE` are likely to be lexed as keyword calls.

Reply:
Corrected.